### PR TITLE
need to convert this to an int

### PR DESCRIPTION
**Pull request by rickardabilia**
I got a silent exception that long can't be cast to int, so I made a conversion in setScreenOfftimeoutHandler

### CODE REVIEW

File: android/src/main/kotlin/com/abilia/system_settings_editor/SystemSettingsHandler.kt [hunk 1]
```diff
@@ -103,9 +103,9 @@ class SystemSettingsHandler(private val context: Context) {
   }
 
   internal fun setScreenOffTimeoutHandler(call: MethodCall, result: MethodChannel.Result) {
-    val timeout: Int? = call.argument("timeout")
+    val timeout: Long? = call.argument("timeout")
     timeout?.let {
-      setScreenOffTimeout(it)
+      setScreenOffTimeout(it.toInt())
```
Comment (bornold) [inline android/src/main/kotlin/com/abilia/system_settings_editor/SystemSettingsHandler.kt]: Couldn't `setScreenOffTimeout`just take a Long now?
Also, increase version number!

Comment (rickardabilia) [reply android/src/main/kotlin/com/abilia/system_settings_editor/SystemSettingsHandler.kt]: Valid. Also forgot to push the tag.

**Review by bornold — APPROVED** [review]
Think tags should be pushed on merge commit from main
no matter, we just rebase this pr onto main

By version i ment pubspec.yml version
And by "just take long" I was wondering if we could 
```kotlin
Settings.System.putLong(
```
instead

Oh well, this is fine, version is just a number anyway

**Comment by rickardabilia** [comment]
No, putLong sadly wasn't possible. Wasn't aware of the pubspec version
________________________________

**Pull request merged**